<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vDesc</class>
 <widget class="QMainWindow" name="vDesc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>vDesc</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="0">
         <widget class="QPushButton" name="changeTokenPb">
          <property name="text">
           <string>Сменить токен</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="nameEdit">
          <property name="placeholderText">
           <string>Здесь будет ваше имя</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLineEdit" name="tokenEdit">
          <property name="placeholderText">
           <string>Ваш токен для аутентификации</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="getFriendsPb">
          <property name="text">
           <string>Список друзей</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="getMessagesPb">
          <property name="text">
           <string>Получить сообщения</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="messagestabWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Друзья</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QTabWidget" name="friendsTabWidget">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="tab_7">
               <attribute name="title">
                <string>Онлайн</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QListWidget" name="friendsOnline"/>
                </item>
                <item>
                 <widget class="QLineEdit" name="messageInput">
                  <property name="placeholderText">
                   <string>Введите текст сообщения</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="sendMessagePb">
                  <property name="text">
                   <string>Отправить сообщение -&gt;</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QTextEdit" name="history">
                  <property name="placeholderText">
                   <string>Здесь будет отображаться история</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_8">
               <attribute name="title">
                <string>Все</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QListWidget" name="allFriends"/>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Сообщения</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QListWidget" name="messages"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>606</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
